assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -38,8 +38,8 @@ transparent_hugepages=never
 
 {{if not .GloballyDisableIrqLoadBalancing}}
 [irqbalance]
-#> Override the value set by cpu-partitioning with an empty one
-banned_cpus=""
+#> Disable the plugin entirely. It can be racy if NTO restarts for whatever reason. We depend now on a systemd unit.
```

Comment:
Same as above NTO restart have nothing to do with this.  `TuneD` restarts do.

Reply:
fixed, thanks.